extensions/vscode-api-tests/package.json

```diff
@@ -41,13 +41,12 @@
 		}
 	},
 	"scripts": {
-		"compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
-		"vscode:prepublish": "node ../../node_modules/gulp/bin/gulp.js --gulpfile ../../build/gulpfile.extensions.js compile-extension:vscode-api-tests ./tsconfig.json"
+		"compile": "gulp compile-extension:vscode-api-tests",
+		"watch": "gulp watch-extension:vscode-api-tests"
```

Comment:
Are you saying it's correct what they are doing? Also note that compilation of extensions happens with this: https://github.com/Microsoft/vscode/blob/master/build/gulpfile.extensions.js#L34. The `watch` and `compile` commands here are merely for illustration or when someone moves out that extensions